src/content/articles/nifty-type-system-explorations.mdx

```diff
@@ -0,0 +1,122 @@
+---
+date: "June 20 2022"
+description: "Some of the most creative, wacky explorations of wha you can do in TypeScript's type system!"
+meta: easter eggs, meta
+---
+
+# Nifty Type System Explorations
```

Comment:
From @ronami: https://gal.hagever.com/posts/typing-the-technical-interview-in-typescript (twitter.com/galstar)